package.json

```diff
@@ -43,7 +43,7 @@
     "@zendeskgarden/react-grid": "8.69.3",
     "@zendeskgarden/react-theming": "8.69.3",
     "@zendeskgarden/react-typography": "8.69.3",
-    "@zendeskgarden/scripts": "1.4.3",
+    "@zendeskgarden/scripts": "2.0.1",
```

Comment:
Does this file need `"type": "module"` similar to [scripts.git](https://github.com/zendeskgarden/scripts/blob/main/package.json#L12)?

Reply:
I don't think we want to release `@zendeskgarden/svg-icons` as an ESM module. The goal of the current task is to incorporate the ESM module `@zendeskgarden/scripts` into each of the repos. Generally speaking, the upgrades will feel more like https://www.gatsbyjs.com/docs/how-to/custom-configuration/es-modules/.